src/wwwroot/js/genpage/main.js

```diff
@@ -2292,6 +2292,17 @@ function storeImageToHistoryWithCurrentParams(img) {
     });
 }
 
+function clearInput() {
```

Comment:
should rename to eg `clearParamFilterInput` ('clearInput' is a little too vague for global namespace)